lib/ansible/modules/files/patch.py

```diff
@@ -100,18 +100,27 @@
 
 import os
 from traceback import format_exc
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import AnsibleModule, get_platform
 from ansible.module_utils._text import to_native
 
 
 class PatchError(Exception):
     pass
 
 
+def add_dry_run_option(opts):
+    # Older versions of FreeBSD, OpenBSD and NetBSD support the --check option only.
```

Comment:
Just checking, you say "older versions", though this seems to be a change for all *BSD operating systems.

Does this need a version check as well?

Will newer versions of *BDD wtill support `--check`, or only `--dry-run`?

Reply:
new versions still support --check for backwards compatibility